<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerWindow</class>
 <widget class="QMainWindow" name="registerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_register_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_register_pwd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_register_pwd2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_register_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_register_pwd1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_register_pwd2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="tip_label2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>64</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 52, 12);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tip_label3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 52, 12);</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="tip_label4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>密码只能含有数字和字母，并且只能以字母开头</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
